### ISSUE TITLE
Duplicate SERVICE-WORKERS reference

### ISSUE THREAD

**Issue by annevk** [tabatkins/bikeshed#1220]
DOM has both SERVICE-WORKERS and SERVICE-WORKERS-2, despite them both being the same URL.

Looks similar to #1196.

I guess there's a bug in specref too that it considers SERVICE-WORKERS to be an alias for -1.

**Comment by tabatkins** [comment]
Nah, unrelated to #1196.  The problem here is that your `<pre class=biblio>` has some terms specified as `spec: SERVICE-WORKERS`, so those terms show up as referencing "SERVICE-WORKERS".  But "ServiceWorkerGlobalScope" is just fetched as normal from the database, where it's recorded as coming from "service-workers-2".

(Because SW-2 isn't in SpecRef, it gets its capitalization in the index and bibliography from the spec db instead, where it's lowercase.)

You can just remove those terms from your `<pre class=biblio>` entirely. The only problem then is that the *correct* definition (in the spec) of "has ever been evaluated flag" is for="script resource", but your biblio block was giving it for="service worker", so you'll get a link failure until you fix that link.  (Lines 1068 and 1112.)

**Comment by tabatkins** [comment]
Yes it does? I'm looking at it right now and it's definitely fixed.

**Comment by annevk** [comment]
https://whatpr.org/dom/634.html#references doesn't have two entries for SERVICE-WORKERS on your end?

**Comment by tabatkins** [comment]
Ah, the reference index is fixed, the biblio still shows both. 

This is because you have explicit biblio references to [SERVICE-WORKERS], which can be found in the specref data, while the linked terms are generating a fake biblio ref for service-workers-2, the spec they link to.

**Comment by annevk** [comment]
1) They end up linking to the same document, so... 2) Versioned references are dumb. I really rather not have to update DOM whenever the W3C decides to issue a new lawyer snapshot.

**Comment by tabatkins** [comment]
1. Yeah, I can probably dedup my "fake" biblio refs if a same-URL SpecRef biblio is in the spec.

2. Sorry man, that's not my issue. W3C publishes leveled specs. You wanna refer to them, you refer to them their way. If you can convince the editors of those specs to do something to accommodate you, great; as long as it's reflected in SpecRef or something like that, I can handle it for you.  I can't special-case these things just because DOM doesn't want to track its dependencies over time. I don't handle RFC obsoletions either.

**Comment by annevk** [comment]
2. Fair, speaking of which, can I have one for Selectors?

**Comment by tabatkins** [comment]
Open an issue in CSSWG. ^_^
